.prettierignore

```diff
@@ -4,3 +4,4 @@ coverage
 client-src/live/web_modules/
 node_modules
 CHANGELOG.md
+package-lock.json
```

Comment:
We can remove this from here, because prettier doesn't change format of package-lock.json

Reply:
> We can remove this from here, because prettier doesn't change format of package-lock.json

I agree.